### DISCUSSION TITLE
Allow localization of Validation Error summary

### DISCUSSION THREAD

**Discussion by dianfishekqi** [laravel/framework#41715] [Ideas]
Hello there,

It would be great if we added localization support to the summeriza part of laravel validation. 

![Screen Shot 2022-03-28 at 14 42 39](https://user-images.githubusercontent.com/4225509/160400226-84fafe00-72cc-4a5b-8f6d-ba33ef836ee4.png)

https://github.com/laravel/framework/blob/f6915333c21b9c656e98f211cf4510115d19c6bd/src/Illuminate/Validation/ValidationException.php#L99

**Comment by schonhoff** [comment]
I'm with you. It is quite ugly if the message is in another language and the ` (and x more errors)` is in English. Even if the message shouldn't be public for the end user it is not nice to read from a dev perspective.
I added a PR for that but it was refused because the message shouldn't be visible for the end user.
https://github.com/laravel/framework/pull/42220
